samples/client/petstore/ruby-faraday/lib/petstore/api/store_api.rb

```diff
@@ -114,20 +117,25 @@ def get_inventory_with_http_info(opts = {})
       # http body (model)
       post_body = opts[:debug_body]
 
-      # return_type
-      return_type = opts[:debug_return_type] || 'Hash<String, Integer>'
-
       # auth_names
       auth_names = opts[:debug_auth_names] || ['api_key']
 
+      # return_type
+      return_type = opts[:debug_return_type]
+
+      return_types_map = {
+        200 => “Hash&lt;String, Integer&gt;“,
```

Comment:
Please use `{{{ ... }}}` instead of `{{ ... }}` to use the original value in the mustache tag.